hs_app_netCDF/models.py

```diff
@@ -163,9 +163,9 @@ def get_hs_term_dict(self):
         # add new terms for NetCDF res
         hs_term_dict["HS_NETCDF_FILE_NAME"] = ""
         for res_file in self.files.all():
-            fn = res_file.resource_file.name
-            if fn.lower().endswith('.nc'):
-                hs_term_dict["HS_NETCDF_FILE_NAME"] = fn[fn.rfind('/')+1:]
+            f_name, f_ext = get_resource_file_name_and_extension(res_file)
+            if f_ext.lower() == '.nc':
+                hs_term_dict["HS_NETCDF_FILE_NAME"] = f_name + f_ext
```

Comment:
@pkdash Say we have a file named "ABC.nc", then the  f_name is "ABC" and the f_ext is ".nc" in this case